docs/faq.rst

```diff
@@ -200,18 +200,25 @@ and call :meth:`aiohttp.web.WebSocketResponse.close` on all of them in
         web.run_app(app, host='localhost', port=8080)
 
 
-How do I make a request from a specific IP address?
----------------------------------------------------
+How do I make a request from a specific IP address or network interface?
+------------------------------------------------------------------------
 
 If your system has several IP interfaces, you may choose one which will
-be used used to bind a socket locally::
```

Comment:
Apparently, both the original and the updated text have the double `used used` bit.